Makefile

```diff
@@ -185,6 +185,11 @@ docopen: out/doc/api/all.html
 docclean:
 	-rm -rf out/doc
 
+run-ci:
+	$(PYTHON) ./configure $(CONFIG_FLAGS)
```

Comment:
@thefourtheye I am assuming it's ok to land as is. Could you give an explicit +1 or -1? Thanks!